src/Provider/RasterProvider.js

```diff
@@ -114,7 +114,7 @@ export default {
         }).then((feature) => {
             if (feature) {
                 layer.feature = feature;
-                layer.extent = layer.feature.extent || layer.feature.geometry.extent;
+                layer.extent = feature.extent || feature.geometry[0].extent;
```

Comment:
Do you think it's possible to have `feature.extent` that would be undefined but feature.geometry[0].extent that would not?

AFAICT both are computed if `options.buildExtent` is true so you could simplify this code.

Reply:
Yep, good catch